references/classification/presets.py

```diff
@@ -1,8 +1,22 @@
 import torch
-from torchvision.transforms import autoaugment, transforms
 from torchvision.transforms.functional import InterpolationMode
 
 
+def get_modules(use_v2):
+    # We need lazy import to avoid the V2 warning in case just V1 is used
+    import torchvision
```

Comment:
Just importing the top namespace shouldn't emit warning, or does it? Plus, since we are not using `torchvision` directly below, but rather import other namespaces, do we need this line at all?

Reply:
I'll move it out and check if we actually need it